contracts/scripts/native_solc_compile_all_ccip

```diff
@@ -78,11 +86,11 @@ compileContract ccip/test/helpers/receivers/MaybeRevertMessageReceiver.sol
 compileContract ccip/test/helpers/MultiOCR3Helper.sol
 compileContract ccip/test/mocks/MockE2EUSDCTokenMessenger.sol
 compileContract ccip/test/mocks/MockE2EUSDCTransmitter.sol
-compileContract ccip/test/WETH9.sol
+# compileContract ccip/test/WETH9.sol (error)
 compileContract ccip/test/helpers/CCIPReaderTester.sol
 
 # Encoding Utils
-compileContract ccip/interfaces/encodingutils/ICCIPEncodingUtils.sol
+# compileContract ccip/interfaces/encodingutils/ICCIPEncodingUtils.sol (error)
```

Comment:
hmm, what is the error here? something we can fix? or is this file not being used by any contract?